packages/aws-cdk/lib/init-templates/v1/app/python/cdk.template.json

```
@@ -1,3 +1,15 @@
 {
-    "app": "%python-executable% app.py"
+    "app": "%python-executable% app.py",
+    "watch": {
+        "include": ["**"],
+        "exclude": [
+          "**/__init__.py",
+          "README.md",
+          "cdk.json",
+          "*.txt",
+          "source.bat",
+          "tests",
+          "python/__pycache__"
```

This should be indented with 4 spaces, not 2.

```suggestion
            "**/__init__.py",
            "README.md",
            "cdk.json",
            "*.txt",
            "source.bat",
            "tests",
          "python/__pycache__"
```